aten/src/ATen/core/stack.h

```diff
@@ -72,11 +87,14 @@ template <typename... Types>
 static inline void pop(Stack& stack, Types&... args) {
   size_t i = 0;
   constexpr size_t N = sizeof...(args);
-  int result[N] = {
+  (void)std::initializer_list<int>{
```

Comment:
This change isn't strictly necessary, right?

Reply:
true, this is something I cleaned up on the way

Reply:
@bhosmer lol remember the msvc issue where we had to land exactly this to fix it? I should have landed this sooner and we would have had to do a lot less debugging there...

Reply:
ugh yeah as you were describing the fix I was thinking of this change and wondering where it had gone 😝